fs/fstest/compare_windows.go

```diff
@@ -19,6 +19,14 @@ package fstest
 // TODO: Any more metadata files generated by Windows layers?
```

Comment:
nit: should this TODO: line be removed now that the additional files are added by this PR?

Reply:
Could argue there still may be more 😅 but I agree, probably time to get rid of this.

Reply:
indeed. There may be more, but we can just add them as we discover them. Removing the TODOs

Reply:
done